modules/nodes-sigstore-using-validation.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-sigstore-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-sigstore-using-validation_{context}"]
+= Validate cluster-wide policies using sigstore
+
+The following procedure demonstrates how to validate cluster-wide policies using Sigstore in {product-title}.
+
+.Prerequisites
+
+* Ensure that the policy on OpenShift image repositories (quay.io/openshift-release-dev/ocp-release, quay.io/openshift-release-dev/ocp-v4.0-art-dev) allows deployment of OpenShift images required for cluster operation.
+
+.Procedure
+
+. Build and Push an Image for Signing:
++
+[source,sh]
+----
+podman login quay.io
+Enter the username and password as prompted:
+Username: qiwanredhat
+Password: <your-password>
+Login Succeeded!
+----
+
+. Build and push the image:
++
+[source,sh]
+----
+echo "FROM scratch" > Dockerfile
+podman build -t quay.io/qiwanredhat/test-sigstoresigned .
+podman push quay.io/qiwanredhat/test-sigstoresigned:latest
+----
+
+. Generate a key pair for cosign:
++
+[source,sh]
+----
+cosign generate-key-pair
```

Comment:
`skopeo generate-sigstore-key`.

AFAIK we don’t ship `cosign` as part of RHEL; if this article should rely on `cosign`, it probably needs to tell users how to get it.